myapp/Dockerfile

```diff
@@ -1,36 +1,25 @@
-FROM ruby:2.6
+FROM ruby:2.6.7
 
 ENV LANG C.UTF-8
 ENV NODE_VERSION 12
 
 RUN apt-get update -qq && \
-    apt-get install -y \
+    apt-get install -qq -y --no-install-recommends \
       build-essential \
-      libpq-dev \
-      nodejs \
       default-mysql-client \
-      jq \
       git \
       curl \
-      python3 \
-      less
+      python3
 
-# node install
-RUN curl -SL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash - \
-    && apt-get install -y nodejs
+# Install Node and Yarn
+RUN curl -SL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash - && \
+    apt-get install -y nodejs && \
+    npm install -g yarn
 
-# yarn install
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    apt-get update && apt-get install -y yarn
+WORKDIR /myapp
 
-ENV APP_ROOT /myapp
-RUN mkdir /$APP_ROOT
```

Comment:
[`WORKDIR`](https://docs.docker.com/engine/reference/builder/#workdir)は存在しないディレクトリーを指定されると自動で作成します。

> If the WORKDIR doesn’t exist, it will be created even if it’s not used in any subsequent Dockerfile instruction.